k8s/jobs/mediawikiUpdate.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -o pipefail
```

Comment:
```suggestion
set -euo pipefail
```

some more bash safety nets:
-e: exit immediately on error
-u: Treat unset variables and parameters as an error

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html